src/openrct2/platform/Platform.Linux.cpp

```diff
@@ -107,39 +85,43 @@ namespace Platform
         auto path = std::string(gCustomOpenrctDataPath);
         if (!path.empty())
         {
-            path = Path::GetAbsolute(path);
+            return Path::GetAbsolute(path);
         }
         else
```

Comment:
Having an else block after an if block that returns is unnecessary.

Reply:
What do you mean? The if block has a condition that isn't always met…

Reply:
Inside the if block it always returns, making it an early exit. The else-block right after if block is not necessary because of that.

Reply:
Ah, right you are, of course.

Reply:
I removed the `else`